templates/commands/render/flags.go

```diff
@@ -115,7 +119,17 @@ func (r *RenderFlags) Register(set *cli.FlagSet) {
 		Target:  &r.Manifest,
 		Default: false,
 		EnvVar:  "ABC_MANIFEST",
-		Usage:   "(experimental) write a manifest file containing metadata that will allow future template upgrades.",
+		// TODO(upgrade): remove "(experimental)"
+		Usage: "(experimental) write a manifest file containing metadata that will allow future template upgrades.",
+	})
+
+	f.BoolVar(&cli.BoolVar{
+		Name:    "manifest-only",
+		Target:  &r.ManifestOnly,
+		Default: false,
+		EnvVar:  "ABC_MANIFEST_ONLY",
+		// TODO(upgrade): remove "(experimental)"
+		Usage: "(experimental) write only a manifest file and no other files; implicitly sets --manifest=true",
 	})
```

Comment:
Do you want to hide these if they are experimental? `Hidden: true`?

Reply:
I did consider that, and I think I want to leave it exposed. If someone wants to be future-proof and generate a manifest file to be ready for the upgrade feature once it exists, I'd like to allow that. And having allowed it, it should appear in the help text (with the `experimental` warning to avoid guaranteeing that the file format won't change).